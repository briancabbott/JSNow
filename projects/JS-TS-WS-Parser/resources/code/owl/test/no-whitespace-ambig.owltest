---
.whitespace
x = y y
y = 'a' | 'a' 'a' | 'aa'
