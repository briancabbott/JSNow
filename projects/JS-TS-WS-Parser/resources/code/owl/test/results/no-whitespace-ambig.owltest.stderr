warning: no whitespace specified - ambiguities may be inaccurate
error: this grammar is ambiguous

. aaa

  can be parsed in two different ways: as

. aa a
  y- y
  x---

  or as

. a aa
  y y-
  x---

